### DISCUSSION TITLE
Export project settings so it can be used to bootstrap other projects, and in CI/CD

### DISCUSSION THREAD

**Discussion by Eggwise** [directus/directus#18498] [Feature Requests]
### Summary

Would be nice if you could export the project with all config like flows, role rules, permissions etc.
This way you can easiliy clone environments and use OTAP, or white label or any multi deployment via CI/CD

### Basic Example

button to export to json or zip or folder.

### Motivation

Would be nice if you could export the project with all config like flows, role rules, permissions etc.
This way you can easiliy clone environments and use OTAP, or white label or any multi deployment via CI/CD

### Detailed Design

todo

### Requirements List

todo

### Drawbacks

todo

### Alternatives

todo

### Adoption Strategy

todo

**Comment by rijkvanzanten** [comment]
This is being tracked in https://github.com/directus/directus/discussions/17495 / https://github.com/directus/directus/discussions/13041 🙂
